src/System.Runtime.Serialization.Primitives/tests/System/Runtime/Serialization/IgnoreDataMemberAttributeTests.cs

```diff
@@ -0,0 +1,16 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.Runtime.Serialization.Tests
```

Comment:
change namespace to ```System.Runtime.Serialization.Primitives.Tests

Reply:
The convention is to namespace the class for the type its testing so this is already correct